src/AppInstallerCLICore/Commands/ConfigureCommand.cpp

```diff
@@ -30,8 +31,9 @@ namespace AppInstaller::CLI
     {
         return {
             // Required for now, make exclusive when history implemented
-            Argument{ Execution::Args::Type::ConfigurationFile, Resource::String::ConfigurationFileArgumentDescription, ArgumentType::Positional, true },
+            Argument{ Execution::Args::Type::ConfigurationFile, Resource::String::ConfigurationFileArgumentDescription, ArgumentType::Positional },
             Argument{ Execution::Args::Type::ConfigurationAcceptWarning, Resource::String::ConfigurationAcceptWarningArgumentDescription, ArgumentType::Flag },
+            Argument{ Execution::Args::Type::ConfigurationEnable, Resource::String::ConfigurationEnableMessage, ArgumentType::Flag, Argument::Visibility::Help },

```

Comment:
With this change, will there also be a GPO to disallow enabling configuration?

Reply:
same as store